lib/fs.js

```diff
@@ -376,6 +376,16 @@ fs.exists = function(path, callback) {
   }
 };
 
+Object.defineProperty(fs.exists, internalUtil.promisify.custom, {
+  enumerable: false,
```

Comment:
All values in `defineProperty` default to `false`, so this may not be necessary?

Alternatively, is it intentional that it is not configurable or writable? (I assume at least the latter is true)